.env.example

```
@@ -21,11 +21,12 @@ LOG_DEPRECATIONS_CHANNEL=null
 LOG_LEVEL=debug
 
 DB_CONNECTION=pgsql
-DB_HOST=127.0.0.1
+DB_HOST=pgsql
 DB_PORT=5432
 DB_DATABASE=laravel
-DB_USERNAME=postgres
-DB_PASSWORD=postgres
+DB_USERNAME=sail
+DB_PASSWORD=password
+DB_PREFIX='dev_'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)